projects/agc-julia-rntuple.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Julia for the Analysis Grand Challenge
+postdate: 2023-04-07
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+experiments:
```

Comment:
Let's add ATLAS here too, the pieces are generic enough that this broadly applies to ATLAS as well.